consensus/consortium/v2/consortium_test.go

```diff
@@ -2417,7 +2411,7 @@ func testIsTrippEffective(t *testing.T, scheme string) {
 		block, _ := core.GenerateChain(&chainConfig, bs[len(bs)-1], ethash.NewFaker(), db, 1, callback, true)
 		bs = append(bs, block...)
 	}
```

Comment:
Could u added some comment explain related to max layers  of Path base then we need to insert bs[399:]?

Reply:
Added